content/articles/multi-label-classification-with-scikit-multilearn/index.md

```diff
@@ -0,0 +1,654 @@
+```
+
+In the above code snippet, we have imported the following.
+
+1. MultinomialNB
+2. accuracy_score
+3. hamming_loss
+4. train_test_split
+5. TfidfVectorizer
+
+#### MultinomialNB
+
+This is a method found in the Naive Bayes algorithm used in building our model, and it is best suited for classification that contains discrete features such a text.
+For detailed understanding about `MultinomialNB`, click [here](https://towardsdatascience.com/naive-bayes-classifier-81d512f50a7c)
+
+#### accuracy_score
+
+This is used to calculate the accuracy of our model when making predictions. The accuracy score is usually expressed in percentages.
+
+#### hamming_loss
+
+This is used to determine the fraction of incorrect predictions of a given model.
+
+#### train_test_split
+
+We use this method to split our dataset into two sets, train set and test set.
+
+#### TfidfVectorizer
+
+It is a statistical measure that evaluates how relevant a word is to a document in a collection of documents. It does this by checking the frequency in which words appear in a certain document. If a word commonly occurs in a document, it is less relevant than a word that is rare in a document.
+
+For detailed understanding about `TfidfVectorizer` click [here](https://medium.com/@cmukesh8688/tf-idf-vectorizer-scikit-learn-dbc0244a911a)
+
+### Installing Scikit-Multilearn
+
+Since we are using Google Colab, we install this library using the following command.
+
+```python
+!pip install scikit-multilearn
+```
+
+This is the library that we will use to perform multi-label text classification. As mention earlier, this tutorial will focus on problem transformation in building our model.
+
+We need to import the required packages to handle the three problem transformation techniques. As explained earlier, the three techniques for problem transformation are binary relevance, classifier chains, and label powerset.
+
+### Importing problem transformation packages
+
+These packages will be used to handle the three techniques.
+
+```python
+from skmultilearn.problem_transform import BinaryRelevance
+from skmultilearn.problem_transform import ClassifierChain
+from skmultilearn.problem_transform import LabelPowerset
+```
+
+### Text preprocessing
+
+Text preprocessing involves removing stop words and removal of noisy data. Stop words are a list of all words that are common in a given language.
+
+Stop words are removed since they do not have a high classification power during predictive analysis. They tend to bring bias when building the classifier.
+
+We remove noisy data that affect our model during training and bring errors in our model.
+
+To perform text preprocessing, we need to install the neat text package. Neat text is a Python package that is used for textual data cleaning and text preprocessing.
+
+#### Neat text installating
+
+Let's install using the following command.
+
+```python
+!pip install neattext
+```
+
+Let's load this package.
+
+#### Loading neat text
+
```

Comment:
I believe neattext is a single word, right? Correct in all areas where you've added space